README.md

```diff
@@ -60,29 +60,25 @@ _Preview means that this is a work in progress and NNCF does not guarantee the f
 
 ### Post-Training Quantization
 
-The NNCF PTQ is the simplest way to apply 8-bit quantization. To run the algorithm you only need your model and a small (~300 samples) calibration no-label dataset. The basic workflow steps:
-1) Load a model and a data source.
```

Comment:
Decided to remove text description of the steps, because the code is self-explanatory